src/ray/common/scheduling/resource_instance_set.cc

```diff
@@ -22,14 +22,22 @@
 namespace ray {
 
 NodeResourceInstanceSet::NodeResourceInstanceSet(const NodeResourceSet &total) {
-  for (auto &resource_id : total.ExplicitResourceIds()) {
+  auto resource_ids = total.ExplicitResourceIds();
+  for (auto &resource_id : resource_ids) {
     std::vector<FixedPoint> instances;
     auto value = total.Get(resource_id);
     if (resource_id.IsUnitInstanceResource()) {
       size_t num_instances = static_cast<size_t>(value.Double());
       for (size_t i = 0; i < num_instances; i++) {
         instances.push_back(1.0);
       };
+    } else if (resource_id == ResourceID::GPU_Memory() &&
+               resource_ids.find(ResourceID::GPU()) != resource_ids.end()) {
+      double num_gpus = total.Get(ResourceID::GPU()).Double();
+      for (size_t i = 0; i < static_cast<size_t>(num_gpus); i++) {
+        instances.push_back(value.Double() / num_gpus);
+      };
```

Comment:
I thought we don't store gpu_memory as node resource?

Reply:
right, forgot to remove